strings/strings.go

```diff
@@ -0,0 +1,13 @@
+// Package strings includes all funcs for manipulating and examining strings
+package strings
+
+// Contains returns a bool indicating if a string array contains a string
+func Contains(s []string, str string) bool {
```

Comment:
Change the name of this function to ExactContains, we'll most likely implement more loose contains k the future

Reply:
done